packages/firebase_core/firebase_core_web/lib/src/firebase_core_web.dart

```diff
@@ -67,7 +67,7 @@ class FirebaseCoreWeb extends FirebasePlatform {
   /// own risk as the version might be unsupported or untested against.
   @visibleForTesting
   String get firebaseSDKVersion {
-    return context['flutterfire_web_sdk_version'] ??
+    return globalContext.getProperty('flutterfire_web_sdk_version'.toJS) ??
```

Comment:
I'm surprised this works without a `toJS` as this returns a `JSAny?` iirc. Should this be `globalContext.getProperty<JSString?>('flutterfire_web_sdk_version'.toJS)?.toDart`? Or perhaps more concisely, `(globalContext['flutterfire_web_sdk_version'] as JSString?)?.toDart`?

Reply:
Good catch, surprinsinlgy, declaring it in a variable triggered an analyzer error that wasn't triggered using `??`
I've updated with your version